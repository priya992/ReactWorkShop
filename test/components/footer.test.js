import React from 'react';
import renderer from 'react-test-renderer';

import Footer from './../../src/components/Footer';

test('Footer component snapshot', () => {
  const component = renderer.create(
    <Footer />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
