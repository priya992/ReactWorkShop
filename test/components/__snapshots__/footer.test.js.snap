// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component 1`] = `
<div>
  Footer Section
</div>
`;
